modules/programs/just.nix

```
@@ -0,0 +1,65 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.just;
+
+in
+{
+  meta.maintainers = [ hm.maintainers.maximsmol ];
+
+  options.programs.just = {
+    enable = mkEnableOption
+      "just - a handy way to save and run project-specific commands";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.just;
+      defaultText = literalExpression "pkgs.just";
+      description = "Package providing the <command>just</command> tool.";
+    };
+
+    enableBashIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to enable Bash integration.
+      '';
+    };
```

All of these enable options can use `mkEnableOption` like so:

```suggestion
    enableBashIntegration = mkEnableOption "Bash integration" // {
      default = true;
    };
```